### ISSUE TITLE
setupClass called for every test if class is not a subclass of TestCase

### ISSUE THREAD

**Issue by MLModel** [nose-devs/nose2#312]
In a test class with no superclasses ```setUpClass``` is executed _once for every test_. If the class is a subclass of ```TestCase``` it is executed only once then the tests are executed.

**Comment by katrinabrock** [comment]
@little-dude is this expected behavior? If so, maybe should be better documented. Even if this is not intended behavior, I'm guessing other users probably rely on it so probably a won't fix, but please weigh in.
